gensim/models/word2vec.py

```diff
@@ -1869,3 +1873,33 @@ def __iter__(self):
         model.accuracy(args.accuracy)
 
     logger.info("finished running %s", program)
+    
+def methodize(func, instance):
+    return MethodType(func, instance, instance.__class__)
+
+
+def load(self, *args, **kwargs):
+    logger.warn('Load was called on instance. Calling on class instead')
+    Word2Vec.load(*args, **kwargs)
```

Comment:
I'd suggest that best would be to error – this method doesn't modify an instance in-place, so the "one and only one right way to do it" is to call it is via the class.